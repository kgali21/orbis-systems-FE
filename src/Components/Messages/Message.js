import React from 'react';
import PropTypes from 'prop-types';
import styles from './Message.module.css';

const Message = ({ tweets, timeStamp }) => (
    <div className={styles.Tweets}>
        <p>{tweets}</p>
        <p className={styles.TimeStamp}>{timeStamp}</p>
    </div>
)

Message.propTypes = {
    tweets: PropTypes.string.isRequired,
    timeStamp: PropTypes.string.isRequired
}

export default Message;