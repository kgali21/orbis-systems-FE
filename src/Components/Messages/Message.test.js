import React from 'react';
import { shallow } from 'enzyme';
import Message from './Message';

describe('Message Component', () => {
    it('renders message', () => {
        const wrapper = shallow(<Message tweets={'test'} timeStamp={'timetest'}/>);
        expect(wrapper).toMatchSnapshot();
    });
});