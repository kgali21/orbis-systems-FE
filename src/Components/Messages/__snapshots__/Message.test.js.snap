// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Component renders message 1`] = `ShallowWrapper {}`;
